repodownloader.py

```diff
@@ -169,6 +170,12 @@ def main(argv):
         default=argv[0],
         help="Subject to prefix notification email with; default: %(default)s",
     )
+    parser.add_argument(
+        "--dirty",
+        action="store_true",
+        type=bool,
+        help="Clean the target directory before fetch and reset",
```

Comment:
```suggestion
        help="Do not clean the target directory before fetch and reset",
```
Since dirty is False by default, the directory gets cleaned if dirty is _not_ specified.